lib/OrdinaryDiffEqStabilizedRK/src/rkc_perform_step.jl

```diff
@@ -1,3 +1,5 @@
+using OrdinaryDiffEq: nlsolve!
```

Comment:
all imports go to the top level as done with the other changes.

Reply:
I am looking at where the tests are failing and adding the imports accordingly